README.md

```diff
@@ -98,16 +98,15 @@ indicated bellow and should work perfectly with similar configurations.
 
 Operating System | Browser | Version
 ---------------- | ------- | -------
-Ubuntu 18.04     | Firefox | 85
-Ubuntu 18.04     | Chrome  | 88
-Windows 10       | Firefox | 85
-Windows 10       | Chrome  | 88
-MacOs            | Firefox | 85
-MacOs            | Chrome  | 88
+Ubuntu 18.04     | Firefox | 88 ?
+Ubuntu 18.04     | Chrome  | 90 ?
```

Comment:
No, we are referring to the version that you will get in those OSs with normal installation.  I will remove '?' since you have verified the versions. I should check the versions on Windows.